addons/stock/models/stock_rule.py

```diff
@@ -221,6 +221,7 @@ def _push_prepare_move_copy_values(self, move_to_copy, new_date):
             'propagate_cancel': self.propagate_cancel,
             'warehouse_id': self.warehouse_id.id,
             'procure_method': 'make_to_order',
+            'description_picking': move_to_copy.product_id.with_context(lang=move_to_copy._get_lang())._get_description(self.picking_type_id),
```

Comment:
```suggestion
            'description_picking': move_to_copy.product_id.with_context(lang=move_to_copy._get_lang())._get_description(self.picking_type_id) or move_to_copy.description_picking,
```
In case we have users trying to propagate a description through all reception steps, they'll still be able to do so if they don't set a specific internal description.